docs/api/clarity.rst

```diff
@@ -123,13 +123,13 @@ Permission
     :show-inheritance:
 
 Placement
-------------
+---------
```

Comment:
This is an rst syntax correction, right? It's supposed to be this way, it's not just a pointless style change? Just checking

Reply:
Yeah, the editor flagged all of the lines where the underline was not the same length as the text. I looked at the others and saw that the majority had a line length matching the text. I assumed this was a copy/paste artifact and added/removed dashes on the ones with warnings.
So lets call it 50% pointless style change and 50% rst syntax correction.